src/Microsoft.AspNetCore.OData/ODataOptions.cs

```diff
@@ -31,5 +31,10 @@ public class ODataOptions
         /// Gets or Sets the set of flags that have options for backward compatibility
         /// </summary>
         public CompatibilityOptions CompatibilityOptions { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether request body binding should be case insensitive.
+        /// </summary>
+        public bool EnableCaseInsensitiveModelBinding { get; set; }
```

Comment:
Name is a bit long, and doesn't really convey that this applies to properties within a request.

Reply:
#Resolved